en/middleware/uxrce_dds.md

```diff
@@ -129,21 +129,21 @@ To build the agent within ROS:
    This builds all the folders under `/src` using the sourced toolchain.
 
 
-To run the XRCE-DDS agent in the workspace:
+To run the micro XRCE-DDS agent in the workspace:
 
 1. Source the `local_setup.bash` to make the executables available in the terminal (also `setup.bash` if using a new terminal).
 
    ```sh
    source /opt/ros/foxy/setup.bash
    source install/local_setup.bash
    ```
-1. Start the agent with settings for connecting to the XRCE-DDS client running on the simulator: 
+1. Start the agent with settings for connecting to the uXRCE-DDS client running on the simulator: 
 
    ```sh
    MicroXRCEAgent udp4 -p 8888
    ```
 
-## Starting XRCE-DDS
+## Starting Agent and Client
```

Comment:
Wasn't sure about whether this is an improvement or not. But I think no-worse. And you've done the work around fixing up the links too, which is good.